cumulusci.yml

```diff
@@ -291,6 +291,20 @@ tasks:
                 STG_SettingsService.stgService.stgCRLP.ContactHardCreditNonSkew_Incremental__c = true;
                 STG_SettingsService.stgService.saveAll();
 
+    deploy_rd2_unmanaged:
+        description: Deploy unmanaged field and layout changes into the org that are needed after enabling RD2 in an org
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            path: unpackaged/config/rd2_post_config
+            namespace_inject: $project_config.project__package__namespace
+
+    validate_rd2_enabled:
```

Comment:
@davisagli  Now that I'm looking at this, I suspect that the actions in this task should be part of the `deploy_rd2_unmanaged` flow, so that it first executes the validation, and if that doesn't throw an exception, it then executes the deployment. Is that correct?

Reply:
You can do that if you want for when you're testing using the CLI, but it'll end up structured a little differently when I configure it for MetaDeploy. Preflight checks get configured separately from flow steps because they run at a different time. i.e. the user clicks the button to run pre-validation checks, then has a chance to review the list of steps for the actual plan (which may have been adjusted based on pre-validation), then clicks the button to run the plan.